src_py/cursors.py

```diff
@@ -365,22 +365,19 @@ def bitswap(num):
 
     info = width, height, hotx, hoty
 
-    for line in range(len(curs)):
-        if curs[line].startswith('static char') or curs[line].startswith('static unsigned char'):
+    possible_starts = ['static char', 'static unsigned char']
+    for i, line in enumerate(curs):
+        if line.startswith(possible_starts):
```

Comment:
TIL: Seems startswith can't be a list, but can be a tuple.

Reply:
Yeap, TIL too :) I fixed it.